frontend/src/components/AuthUtils/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Utility functions to get various parameters corresponding to the currently
+ * authenticated user. These functions should ONLY be called from private pages,
+ * i.e. the user must be authenticated to ensure the parameters can be
+ * retrieved.
+ */
+
+import { AuthUtilsConsumer, getCurrentUser } from './AuthUtilsConsumer.js';
+import { SIGN_IN } from '../../constants/routes.js';
+
+/**
+ * Called when one of the authentication utility functions are used when the
+ * user has not logged in yet. Since the utility functions require that they be
+ * called from private pages, this function being called implies that the user
+ * managed to access a private page without being authenticated first. So they
+ * are immiedately redirected to the SIGN_IN page.
+ */
+function redirectToSignIn() {
+  window.location.href = SIGN_IN;
+}
+
+/**
+ * @returns {String} The user's display name.
+ */
+export function getUserDisplayName() {
+  if (!getCurrentUser()) {
```

Comment:
Should something more robust be used here? Example: https://stackoverflow.com/questions/5113374/javascript-check-if-variable-exists-is-defined-initialized

Alternatively, see my comment below in the testing section. The initial value could be defined to ensure this is safe.

Reply:
I decided to explicitly check for null instead of relying on JavaScript truthiness. 👍